js/upload.js

```diff
@@ -15,7 +15,9 @@
     xhr.addEventListener('timeout', function () {
       onError('Запрос не успел выполниться за ' + xhr.timeout + 'мс');
     });
-
+    xhr.addEventListener('error', function () {
```

Comment:
можно без функции-обертки
```suggestion
    xhr.addEventListener('error', onError)
```